go.mod

```diff
@@ -134,3 +135,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.5.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/rhobs/observability-operator/pkg/apis v0.0.0-unpublished => ./pkg/apis
```

Comment:
It will probably be better to do this in a follow-up PR (when the module exists in the repository)